src/common/commands_parser.rs

```diff
@@ -163,6 +163,8 @@ impl TerminateEnclavesArgs {
 pub struct ConsoleArgs {
     /// The ID of the enclave whose console is to be shown.
     pub enclave_id: String,
+    /// The time in seconds after the console disconnects from the enclave.
+    pub disconnect_timeout: Option<u64>,
```

Comment:
then let's call it disconnect_timeout_ms.

Reply:
why ms? the time is measured in seconds

Reply:
seconds :)

Reply:
no, this was not resolved disconnect_timeout_sec.